DistributionPackages/Neos.NeosIo.ServiceOfferings/Configuration/NodeTypes.Document.ServiceProvider.yaml

```diff
@@ -98,11 +98,16 @@
             values:
               '1':
                 label: 'Single person / Freelancer'
+                icon: user
```

Comment:
Do this icons work? Don't it have to be `fas fa-user`?

Reply:
Sure, since a while you don't need a prefix anymore.

Reply:
Wow, didn't know that. Good to know